app/models/gi_bill_feedback.rb

```diff
@@ -28,11 +28,10 @@ def get_user_details
     profile_data = {}
 
     if user.present?
```

Comment:
The assignment to profile_data seems unecessary when looking at this deeper. And I'm a big fan of the pattern of just returning before the if condition. Something like this:

```
def get_user_details 
  return unless user.present?
  { profile_data: { 'active_ICN'     => user.icn,
                            'historical_ICN' => user.historical_icns,
                           'sec_ID'             => user.sec_id } }
end
```

Reply:
Ah yeah, that could work, although it would be just a bit different, since it's not actually returning nil is the user isn't present:
`{ 'profile_data' => {} } unless user.present?`

Reply:
I guess the original thinking was probably that the last line would show that the function returns a hash with `profile_data` key regardless of the presence of `User`